src/deep-core/lib/AWS/Lambda/Runtime.js

```diff
@@ -99,11 +108,15 @@ export class Runtime extends Interface {
   /**
    * @param {Object} event
    * @param {Object} context
+   * @param {Function} lambdaCallback
    * @returns {Runtime}
    */
-  run(event, context) {
+  run(event, context, lambdaCallback) {
     this._context = new Context(context);
     this._request = new Request(event);
+    lambdaCallback && (this._resolver = new Resolver(lambdaCallback));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/MitocGroup/deep-framework/file/2442937790/issues/source?bid=2895475&fileBranchId=3245548#l117)